resources/css/filament/admin/theme.css

```diff
@@ -7,3 +7,106 @@
 .min-w-kanban {
     min-width: 20rem;
 }
+
+/* Bright Sun */
+.dark {
+    --primary-50: 255, 251, 235;
+    --primary-100: 255, 244, 198;
+    --primary-200: 254, 233, 137;
+    --primary-300: 254, 212, 63;
+    --primary-400: 254, 195, 33;
+    --primary-500: 248, 162, 8;
+    --primary-600: 219, 122, 4;
+    --primary-700: 182, 86, 7;
+    --primary-800: 148, 66, 12;
+    --primary-900: 121, 55, 14;
+    --primary-950: 70, 27, 2;
+}
+
+/* Style Overrides */
```

Comment:
Do we want to keep this commented out code for now?

Reply:
The overrides? Yes. I've brought up the conversation that we had yesterday with Product in the ticket, and we'll see which way we need to go. 

With the approach taken, the overrides are mostly redundant, but I'm leaving them here in the event that we do want to introduce overrides in specific cases.